apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx

```diff
@@ -170,12 +173,13 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
         eachPropertyMeta: CommonPluggableComponentMetaPropertyInterface,
         isSub?: boolean,
         testId?: string,
-        listen?: (key: string, values: Map<string, FormValue>) => void):
+        listen?: (key: string, values: Map<string, FormValue>) => void,
+        showField?: boolean):
         ReactElement => {
 
         if (isSub) {
             return (
-                <Grid.Row columns={ 2 } key={ eachPropertyMeta?.displayOrder }>
+                showField && (<Grid.Row columns={ 2 } key={ eachPropertyMeta?.displayOrder }>
```

Comment:
Shall we move this condition to L180 to enhance readabiilty?

Reply:
This is to maintain consistency.